src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsCreatorInterface.php

```diff
@@ -18,4 +18,11 @@ interface UnzerCredentialsCreatorInterface
      * @return \Generated\Shared\Transfer\UnzerCredentialsResponseTransfer
      */
     public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer;
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsResponseTransfer
+     */
+    public function createUnzerCredentialsAndSetUnzerNotificationUrl(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer;
```

Comment:
```suggestion
    public function createUnzerCredentialsAndSetUnzerNotificationUrl(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer;
```

The naming alone indicates that this should be split into two methods. We never use "and", what if the internal behavior changes? What if we suddenly need xAndYAndz? I would suggest to split this in createUnzerCredentials() and setUnzerNotificationUrl().